Constitution.txt

```diff
@@ -1,490 +1,264 @@
 
-2: The Privilege of the Writ of Habeas Corpus shall not be suspended, unless when in Cases of Rebellion or Invasion the public Safety may require it.
+2: No Bill of Attainder or ex post facto Law shall be passed.
 
-3: No Bill of Attainder or ex post facto Law shall be passed.
+3: No Capitation, or other direct, Tax shall be laid.
 
-4: No Capitation, or other direct, Tax shall be laid, unless in Proportion to the Census or Enumeration herein before directed to be taken.7
+4: No Tax or Duty shall be laid on Articles exported from any State, unless such Tax or Duty shall be imposed uniformly upon all States.
 
-5: No Tax or Duty shall be laid on Articles exported from any State.
+5: No Preference shall be given by any Regulation of Commerce or Revenue to the Ports of one State over those of another: nor shall Vessels bound to, or from, one State, be obliged to enter, clear, or pay Duties in another.
 
-6: No Preference shall be given by any Regulation of Commerce or Revenue to the Ports of one State over those of another: nor shall Vessels bound to, or from, one State, be obliged to enter, clear, or pay Duties in another.
+6: No Money shall be drawn from the Treasury, but in Consequence of Appropriations made by Law; and a regular Statement and Account of the Receipts and Expenditures of all public Money shall be published at least once per year. Such Statement shall include a record of any debts or surpluses arising from any individual Bill.
 
-7: No Money shall be drawn from the Treasury, but in Consequence of Appropriations made by Law; and a regular Statement and Account of the Receipts and Expenditures of all public Money shall be published from time to time.
+7: No Title of Nobility shall be granted by these United States: And no Person holding any Office of Profit or Trust under them, shall, without the Consent of the Congress, accept of any present, Emolument, Office, or Title, of any kind whatever, from any King, Prince, or foreign State. No person holding a Title of Nobility in a State shall have any authority relating to his Title in any other State, nor shall he have any authority over the Union.
 
-8: No Title of Nobility shall be granted by the United States: And no Person holding any Office of Profit or Trust under them, shall, without the Consent of the Congress, accept of any present, Emolument, Office, or Title, of any kind whatever, from any King, Prince, or foreign State.
+8: No law, increasing the compensation for the services of the Senators and Representatives, shall take effect upon a Senator or Representative who shall be, at time of passage of the law, in the respective office; nor shall any law, decreasing their compensation, take effect until a new election shall have intervened.
```

Comment:
Here I incorporate the 27th amendment, with minor updates: A given congressman cannot vote for himself to receive a raise, ever; he can only vote to give his successor a raise. He can, however, vote to give himself a pay cut, taking effect after an election.